src/components/Card/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import styles from './styles.module.scss';
+
+interface CardProps {
+  source: string;
+  alt: string;
+  title: string;
+  subtitle: string;
+}
+
+function Card({ source = '', alt = 'no-image', title, subtitle }: CardProps) {
```

Comment:
Why the default alt is no-image?
And check that your CardProps says that "alt" prop is required.